util/util.go

```diff
@@ -350,3 +354,73 @@ func WriteWireGuardServerConfig(tmplBox *rice.Box, serverConfig model.Server, cl
 
 	return nil
 }
+
+// RestartWireGuardProces to restart wireguard interface when a new client is added or deleted
+func RestartWireGuardProces(globalSettings model.GlobalSetting) error {
+	ss := strings.Split(globalSettings.ConfigFilePath, "/")
+	wireguardInterfaceName := strings.Split(ss[len(ss)-1], ".")[0]
+	app := "wg-quick"
+
+	cmd := exec.Command(app, "up", wireguardInterfaceName)
+	_, err := cmd.Output()
+
+	if err != nil {
+		return err
+	}
+
+	cmd = exec.Command(app, "down", wireguardInterfaceName)
+	_, err = cmd.Output()
+
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// SendMailToNewUser Send mail to newly created user
+func SendMailToNewUser(client *model.Client) error {
+	m := mail.NewV3Mail()
+
+	from := mail.NewEmail("Wireguard", SendGridUser)
+	to := mail.NewEmail(client.Name, client.Email)
+
+	m.SetFrom(from)
+
+	personalization := mail.NewPersonalization()
+	personalization.AddTos(to)
+	personalization.Subject = "[CybelAngel] Your access to wireguard"
+
+	// add `personalization` to `m`
+	m.AddPersonalizations(personalization)
+
+	htmlContent := mail.NewContent(
+		"text/html",
+		"<strong>Please find attach your wireguard configuration</strong>",
```

Comment:
It's a little bit light :) 

Maybe we should tell something about how to setup this configuration? With a reference to the workstation-playbook which will setup the Wireguard client and/or the standart wireguard client configuration